.github/workflows/release.yml

```diff
@@ -0,0 +1,61 @@
+name: Release
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 1-7 3,6,9,12 *"
+    
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Only run on mondays
+        id: is-it-monday
+        continue-on-error: true
+        run: |
+          if [[ ! $(date +%u) -eq 1 ]]; then
```

Comment:
Hmm, seems slightly strange for this to be 1-indexed

I'll take your word that this detects Mondays

Reply:
https://man7.org/linux/man-pages/man3/strftime.3.html

```
%u     The day of the week as a decimal, range 1 to 7, Monday
       being 1.  See also %w.  (Calculated from tm_wday.)  (SU)

%w     The day of the week as a decimal, range 0 to 6, Sunday
       being 0.  See also %u.  (Calculated from tm_wday.)
```

Reply:
Yeah, I don't doubt it's correct.

I'm just amused by it, since ime the correct behaviour is 0-6, Monday to Sunday.